rllib/models/tf/tf_action_dist.py

```diff
@@ -135,6 +138,8 @@ def logp(self, actions: TensorType) -> TensorType:
             if isinstance(self.action_space, gym.spaces.Box):
                 actions = tf.reshape(
                     actions, [-1, int(np.product(self.action_space.shape))])
+            elif isinstance(self.action_space, gym.spaces.MultiDiscrete):
```

Comment:
Bug fix.

Reply:
how did you discover this bug?
can we make it a unit test for tf_action_dist?

Reply:
Yeah, great question. True, this should be tested more exhaustively on the unit test level (distribution tests).